application/classes/Ushahidi/Console/Webhook.php

```diff
@@ -0,0 +1,135 @@
+<?php defined('SYSPATH') or die('No direct script access');
+
+/**
+ * Ushahidi Webhook Console Command
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Console\Command;
+
+use Ushahidi\Core\Tool\Signer;
+use Ushahidi\Core\Entity\PostRepository;
+use Ushahidi\Core\Entity\WebhookQueueRepository;
+use Ushahidi\Core\Entity\WebhookRepository;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Request;
+
+class Ushahidi_Console_Webhook extends Command
+{
+	private $db;
+	private $postRepository;
+	private $webhookRepository;
+	private $webhookQueueRepository;
+	private $signer;
+	private $client;
+
+	public function setDatabase(Database $db)
+	{
+		$this->db = $db;
+	}
+
+	public function setWebhookRepo(WebhookRepository $repo)
+	{
+		$this->webhookRepository = $repo;
+	}
+
+	public function setPostRepo(PostRepository $repo)
+	{
+		$this->postRepository = $repo;
+	}
+
+	public function setSigner(Signer $signer)
+	{
+		$this->signer = $signer;
+	}
+
+	public function setWebhookQueueRepo(WebhookQueueRepository $repo)
+	{
+		$this->webhookQueueRepository = $repo;
+	}
+
+	protected function configure()
+	{
+		$this
+			->setName('webhook')
+			->setDescription('Manage webhook requests')
+			->addArgument('action', InputArgument::OPTIONAL, 'list, send', 'list')
+			->addOption('limit', ['l'], InputOption::VALUE_OPTIONAL, 'number of webhook requests to be sent')
+			;
+	}
+
+	protected function executeList(InputInterface $input, OutputInterface $output)
+	{
+		return [
+			[
+				'Available actions' => 'send'
+			]
+		];
+	}
+
+	protected function executeSend(InputInterface $input, OutputInterface $output)
+	{
+
+		$this->client = new GuzzleHttp\Client();
+
+		$limit = $input->getOption('limit');
+
+		$count = 0;
+
+		// Get Queued webhook requests
+		$webhook_requests = $this->webhookQueueRepository->getWebhooks($limit);
+
+		// Start transaction
+		$this->db->begin();
+
+		foreach ($webhook_requests as $webhook_request) {
+			// Generate request
+			$request = $this->generateRequest($webhook_request);
+			$this->client->sendAsync($equest);
```

Comment:
Could we just move the send inside `generateRequest` ie. change it to processRequest() or something